.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: c
+dist: trusty
+
+matrix:
+  include:
+  - env: DISTRO=ubuntu IMAGE=ubuntu:xenial
```

Comment:
Any distro is possible (any "big" released distro, at least). I started with F25 and Ubuntu LTS. But, I saw some problems on Fedora. 
* I just focused on a working solution.
* We can insert Fedora/Debian and start to fix/workaround bugs.